packages/svelte/package.json

```diff
@@ -29,7 +29,7 @@
     "!dist/**/*.spec.*"
   ],
   "peerDependencies": {
-    "svelte": "^4.0.0",
+    "svelte": "^4.0.0 || ^5.0.0-next.0",
```

Comment:
When is this expected to land as stable? I don't know many libs that include prereleases in their peer dep range (e.g React 19 and Next 15 you just have to deal with 100 packages yelling).

Also since none of us are familiar with Svelte we don't wanna bump this and have an expectation to fix it if they introduce some breaking change before official release

Reply:
Ok 👍 I'll let @markflorkowski have the final say